.gitignore

```diff
@@ -0,0 +1,4 @@
+nbproject
+
+/tests/E2E/
+/vendor/
```

Comment:
why `nbproject`? Should we add something like https://github.com/github/gitignore/blob/master/Prestashop.gitignore? or at least:

- .project
- .idea
- build
- coverage
- .env